pages/api/webhooks/discord.ts

```diff
@@ -0,0 +1,45 @@
+import ContactInformationProps from '../../../interfaces/ConctactInformationProps';
+import ServiceInformationFormProps from '../../../interfaces/ServiceInformationFormProps';
+
+import {
+    NEW_CONSULTATION,
+    NAME,
+    EMAIL,
+    PHONE_NUMBER,
+    MAX_BUDGET,
+    MIN_BUDGET,
+    COMPANY_TYPE,
+    PROJECT_GOALS,
+    TIMELINE_BUDGET_CONSTRAINTS
+} from '../../../constants/strings'
+
+const { Webhook, MessageBuilder } = require('discord-webhook-node');
+
+const hook = new Webhook(process.env.DISCORD_WEBHOOK);
+
+export default async function sendMessage(
+    conctactInformation: ContactInformationProps, 
+    budget: any, 
```

Comment:
Let's add a type for this `BudgetFormProps`

```
type BudgetFormProps = {
  minBudget: number,
  maxBudget: number
}
```

Reply:
Should I create a new file in the system called "types" where we store this ? Or should I do it how we usually do it with `interface` ?

Reply:
probably just thesame place as `ContactInformationProps`